### DISCUSSION TITLE
[@types/video.js] VideoJsPlayerOptions is missing id property

### DISCUSSION THREAD

**Discussion by BrainCrumbz** [DefinitelyTyped/DefinitelyTyped#59436] [Issues with a @types package] [answered]
Hi all

we opened an [issue](https://github.com/videojs/video.js/issues/7670) on video.js repo, which confirms that such `id` property is present and used in current source code, but is missing both in documentation and type definition, and it should exist in both of them.

In effect, looking at current version of [video.js/index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7442d1be1371abb6ec2956f4e6b94d547a8ba712/types/video.js/index.d.ts#L7026), we could not see the property defined.

[a comment by an automated account is not shown]

**Comment by joeflateau** [comment]
Once Video.js has documented the option then it can be added to the type definitions. These type definitions should not provide types for undocumented properties.

**Comment by peterblazejewicz** [comment, marked as answer]
I've created request, they coded implementation wrong, not every element is static. I have the same implementation for players (cctv), though we are not using types (yet)
#59437
